queries.md

```diff
@@ -8,72 +8,107 @@
 
```

Comment:
Missing the first question:

**All the companies that it's name match 'Babelgum'. Retrieve only their name field.**